charts/argocd-apps/values-test.yaml

```diff
@@ -36,3 +36,47 @@ applications:
       value: "1"
     - name: worker.replicas
       value: "1"
+- name: router
+  helm:
+    parameters:
+    - name: govukEnvironment
+      value: "test"
```

Comment:
It's a shame that this way we end up with the `{name: "foo", value: "bar"}` antipattern rather than `{foo: "bar"}`, times hundreds of config items times 3-4 environments. Maybe it's the lesser of evils for now?

I'm starting to think that once Argo starts supporting reading Helm values files from alternative repos we might want to switch back to using ordinary values files for these configs, which would be way more succinct and readable. It'd also neatly solve the problem that all the apps' configs are lumped together in one file rather than separated by app (which isn't a problem for now, but might potentially become one eventually depending on whether the restructured dev teams are happy with sharing config files).